docs/install.rst

```diff
@@ -1,26 +1,126 @@
+The easiest way to manage a Python installation together with an Astropy installation is with the ``conda`` package manager.
+If you already have Python and ``conda`` working you can skip the next section.
+However, if you are using the Anaconda Python Distribution, we recommend you still install miniforge as described below.
 
-Using pip
-=========
+Installing miniforge
+====================
+
+We recommend installing Python with `miniforge <https://github.com/conda-forge/miniforge/#miniforge>`__.
+This will install ``conda`` and automatically configure the default channel (a channel is a remote software repository) to be ``conda-forge``.
+
+First, download the installer for your system and architecture from the links below:
+
+.. grid:: 3
+
+    .. grid-item-card:: Linux
+
+        `x86-64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh>`__
+
+        `aarch64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh>`__
+
+        `ppc64le <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-ppc64le.sh>`__
+
+    .. grid-item-card:: Windows
+        :link: https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe
+
+        `x86-64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe>`__
+
+    .. grid-item-card:: Mac
+
+        `x86-64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh>`__
+
+        `arm64 (Apple Silicon) <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh>`__
+
+Then select your platform to install miniforge:
+
+.. tab-set::
+
+    .. tab-item:: Linux & Mac
+        :sync: platform
+
+        Linux & Mac Run the script downloaded above, with
+        ``bash <filename>``. The following should work:
+
+        .. code-block:: console
+
+            bash Miniforge3-$(uname)-$(uname -m).sh
+
+        Once the installer has completed, close and reopen your terminal.
+
+    .. tab-item:: Windows
+        :sync: platform
+
+        Double click the executable file downloaded from
+        the links above.
+
+        Once the installer has completed you should have a new "miniforge
+        Prompt" entry in your start menu.
+
+In a new terminal (miniforge Prompt on Windows) run ``conda list`` to test that the install has worked.
+
+Using an Existing Conda Installation
+====================================
+
+If you already have a conda installation you can use it to install astropy.
+We do not recommend reusing an anaconda installation, as the default packages provided by Anaconda Inc. are often out dated and can require a license for use at a large organisaiton.
+If you have installed miniforge or anaconda you can add ``-c conda-forge`` to all installation commands to install packages from `conda-forge <https://conda-forge.org>`__.
```

Comment:
Maybe a good place to also mention how to do it permanently ?
```
conda config --prepend channels conda-forge
```

Reply:
I got myself stuck in a bit of a brain loop with this. You could write a whole essay on how to unravel yourself out of a defaults / anaconda install and not get shot in the foot.

sunpy has [this](https://docs.sunpy.org/en/stable/tutorial/installation.html#installing-sunpy) here:
```
conda config --add channels conda-forge
conda config --set channel_priority strict
```
which is a good thing to have, but if you have installed miniforge then you don't need to run it, and if you have full-fat anaconda then running them has the potential to really break your base environment. So I am not sure what the best thing to recommend to people who really want to use Anaconda or don't want to install a new miniforge.

Reply:
Right. We should probably keep it simple and repeatable then. Users who get tired of typing `-c conda-forge` can figure out how to skip it easily enough.